net/nimble/host/mesh/src/net.c

```diff
@@ -592,7 +594,7 @@ void bt_mesh_iv_update(u32_t iv_index, bool iv_update)
 		}
 	}
 
-	if (!(MYNEWT_VAL(BLE_MESH_IV_UPDATE_TEST))) {
+	if (!iv_update_test_mode) {
```

Comment:
maybe instead of this we could have some macro in pts.h which would take decision based on (MYNEWT_VAL(BLE_MESH_IV_UPDATE_TEST) and iv_update_test_mode. BTW iv_update_test_mode we could move to pts.c then.